src/server/JasmineGraphInstanceService.cpp

```diff
@@ -80,4301 +144,5243 @@ void *instanceservicesession(void *dummyPt) {
-            }
+                bzero(data, INSTANCE_DATA_LENGTH + 1);
+                int result_wr = write(sockfd, JasmineGraphInstanceProtocol::HANDSHAKE.c_str(),
+                                      JasmineGraphInstanceProtocol::HANDSHAKE.size());
 
-            graphDB = graphDBMapLocalStoresPgrnk[graphID + "_" + partitionID];
-            centralDB = graphDBMapCentralStores[graphID + "_centralstore_" + partitionID];
-
-            map<long, map<long, unordered_set<long>>> egonetMap =
-                calculateLocalEgoNet(graphID, partitionID, serverPort, graphDB, centralDB, workerSockets);
-
-            string instanceDataFolderLocation =
-                Utils::getJasmineGraphProperty("org.jasminegraph.server.instance.datafolder");
-            string attributeFilePart = instanceDataFolderLocation + "/" + graphID + "_egonet_" + partitionID;
-            ofstream partfile;
-            partfile.open(attributeFilePart, std::fstream::trunc);
-            for (map<long, map<long, unordered_set<long>>>::iterator it = egonetMap.begin(); it != egonetMap.end();
-                 ++it) {
-                map<long, unordered_set<long>> egonetInternalMap = it->second;
-                for (map<long, unordered_set<long>>::iterator itm = egonetInternalMap.begin();
-                     itm != egonetInternalMap.end(); ++itm) {
-                    unordered_set<long> egonetInternalMapEdges = itm->second;
-                    for (unordered_set<long>::iterator ite = egonetInternalMapEdges.begin();
-                         ite != egonetInternalMapEdges.end(); ++ite) {
-                        partfile << to_string(it->first) << "\t" << to_string(itm->first) << "\t" << to_string(*ite)
-                                 << endl;
-                    }
+                if (result_wr < 0) {
+                    instance_logger.log("Error writing to socket", "error");
                 }
-            }
-            partfile.close();
 
-            instance_logger.log("Egonet calculation complete", "info");
-
-        } else if (line.compare(JasmineGraphInstanceProtocol::TRIANGLES) == 0) {
-            instance_logger.log("Received : " + JasmineGraphInstanceProtocol::TRIANGLES, "info");
-            write(connFd, JasmineGraphInstanceProtocol::OK.c_str(), JasmineGraphInstanceProtocol::OK.size());
-            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::OK, "info");
-            bzero(data, INSTANCE_DATA_LENGTH + 1);
-            read(connFd, data, INSTANCE_DATA_LENGTH);
-            string graphID = (data);
-            graphID = Utils::trim_copy(graphID, " \f\n\r\t\v");
-            instance_logger.log("Received Graph ID: " + graphID, "info");
+                instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::HANDSHAKE, "info");
+                bzero(data, INSTANCE_DATA_LENGTH + 1);
+                read(sockfd, data, INSTANCE_DATA_LENGTH);
+                string response = (data);
```

Comment:
I do not see a point of having parenthesis around the variable data.

Reply:
All similar occurrences are removed in next PR

Reply:
Ok